tests/Api/PersistentSpec.hs

```diff
@@ -169,3 +176,13 @@ spec = do
                     it "will indeed log you in (yeay)" $ \(Nat rp) -> do
                         user:_ <- liftIO $ rp getUsers
                         t rp (user ^. userLogin) isJust
+
+    regression
+
+-- * Regression suite
+
+regression :: Spec
+regression = context "regression" $ do
```

Comment:
`context` is specifically for system state ("with database empty"; "with user logged out"; ...), and in this place I would like to see `describe`.

Also, can you add an actual description?  "regression" doesn't count.  (-: